### PR TITLE
ILabs port from LIMS-Scripts package

### PR DESCRIPTION
**Pull request by dmcmanam**
To deploy we need to add two values to app.properties and then change the crontab on igolims.

### CODE REVIEW

File: src/main/java/org/mskcc/limsrest/service/ilabs/Researcher.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package org.mskcc.limsrest.service.ilabs;
+
```
Comment (FahimehMirhaj) [inline src/main/java/org/mskcc/limsrest/service/ilabs/Researcher.java:2]: Adding some class description improves the readability.

File: src/main/java/org/mskcc/limsrest/service/ilabs/GetGeneralInfo.java [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+    public void printMapping(Map<String, String> ilabValues) {
+        for (Map.Entry<String, String> entry : ilabValues.entrySet()) {
+            System.out.println(entry.getKey() + ":");
+            System.out.println("  " + entry.getValue());
+        }
+        System.out.println();
+
+    }
+
+    @Deprecated
```
Comment (FahimehMirhaj) [inline src/main/java/org/mskcc/limsrest/service/ilabs/GetGeneralInfo.java:304]: Do we need this deprecated function?

File: src/main/java/org/mskcc/limsrest/service/ilabs/CustomForm.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomForm {
+    private String id;
+    private String name;
+    private String note;
+    private Map<String, String> fields;
+
+    public CustomForm() {
```
Comment (FahimehMirhaj) [inline src/main/java/org/mskcc/limsrest/service/ilabs/CustomForm.java:12]: For pulling iLab to Teamwork cards I've already added this class under [src/main/java/org/mskcc/limsrest/service/]. So, this is redundant.

**Review by FahimehMirhaj — CHANGES_REQUESTED** [review]
We can test on dev server before moving to prod.

File: src/main/java/org/mskcc/limsrest/service/ilabs/CustomForm.java (hunk 3 above)
Comment (dmcmanam) [reply src/main/java/org/mskcc/limsrest/service/ilabs/CustomForm.java:12]: ok

File: src/main/java/org/mskcc/limsrest/service/ilabs/GetGeneralInfo.java (hunk 2 above)
Comment (dmcmanam) [reply src/main/java/org/mskcc/limsrest/service/ilabs/GetGeneralInfo.java:304]: This function has old unused Ruby code so I removed it.

**Review by FahimehMirhaj — APPROVED** [review]
Looks good to me.

**Pull request merged**